src/components/PeopleFilters.tsx

```diff
@@ -1,27 +1,65 @@
-export const PeopleFilters = () => {
+import cn from 'classnames';
+import { SexType } from '../types/SexType';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  query: string;
+  sex: SexType;
+  centuries: string[];
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  // handleSexChange: (sex: SexType) => void;
+  // toggleCenturies: (ch: string) => void;
```

Comment:
The handlers `handleSexChange` and `toggleCenturies` are commented out. If these are necessary for the component's functionality, consider implementing and using them.